EXIOBASE_conversion_software/csv2rdf.py

```diff
@@ -5,10 +5,10 @@
 # This is a specific solution. Other uses need to adapt accordingly
 #
 # Last modified Time-stamp: <2018-12-14 10:43:21 CET (vle)>
-# Author: Vang Le<vle@its.aau.dk>
-# Created: 2018-12-13T21:34:06+0100
+# Author: Emil Riis Hansen <emilrh@cs.aau.dk>, Matteo Lissandrini <matteo@cs.aau.dk>
```

Comment:
@agneta20  This code is based on code by Vang Le, so I think it would be fair to keep the full list of authors and the original copyright date